plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -64,11 +64,20 @@ To determine the position of the outputs of a transaction, you will include the
 
 
 
-## Transaction type and output type
+## Transaction type, output type, and spending condition
 
-ALD introduces the concepts of transaction type, and transcation output type. Each transaction type and transcation output type can define different rules about how to spend funds.
+ALD introduces the concepts of transaction type and transaction output type.
+Each transaction output type can define its own rule about how funds may be spent.
```

Comment:
`its own set of rules`

And then I'd add more explanation of the concept of a spending condition, before diving into links to code and "gory details" like the `hash` part. Like what is the abstraction, that a spending condition is defined per the pair `tx_type` `output_type` and so forth.

Reply:
I've provided an example of how spending condition can be used, before the explanation of how it is implemented.

Reply:
hm, the example is nice (consider choosing sth less speculative though, like a EC signature, which you can then link to our existing Payment tx spending condition). Also mind to keep it to a separate paragraph (check rendered `.md`)

However, I'd insist on providing some not-an-example explanation of a spending condition concept. Just explain what it is, in 2-3 sentences, one of them could literally be "a spending condition is defined per the pair of a transaction type and an output type".